prosody/rootfs/defaults/prosody.cfg.lua

```diff
@@ -173,7 +171,8 @@ network_settings = {
   tcp_backlog = 511;
 }
 
-component_interface = { "*" }
+http_ports = { 5280 }
+http_interfaces = { "*", "::" }
```

Comment:
Do we need this for jitsi-meet default deb installation?

Reply:
No because we proxy to localhost.